### DISCUSSION TITLE
Nova-2 vs Nova-2-medical

### DISCUSSION THREAD

**Discussion by scoutaroom** [deepgram/community#566] [General help]
We need your help . 
What are diarization capabilites of models nova-2 and nova-2-medical we have some cases in which nova-2-medical doesn't recognize another speaker even if there is 0.8s or higher break between speakers. We think this issue is not related to multichannel, because all words are detected.
Is this issue could be related to audio file format or audio quality?
What options should we use improve diarization?
We use transcribeFileCallback API call with options:
model: "nova-2-medical",
smart_format: true,
diarize: true,

API call 
prerecorded.transcribeFileCallback

model: "nova-2-medical",
smart_format: true,
diarize: true,

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.
